conda_self/cli/main_remove.py

```diff
@@ -10,10 +10,21 @@
 
 def configure_parser(parser: argparse.ArgumentParser) -> None:
     parser.description = HELP
-    parser.add_argument("specs", nargs="+", help="Plugins to install")
+    parser.add_argument("specs", nargs="+", help="Plugins to remove/uninstall")
     parser.set_defaults(func=execute)
 
 
 def execute(args: argparse.Namespace) -> int:
+    from ..query import permanent_dependencies
+    from ..install import uninstall_specs_in_protected_env
+
+    uninstallable_packages = permanent_dependencies()
+    for spec in args.specs:
+        if spec in uninstallable_packages:
+            print(f"Package '{spec}' can not be removed. Aborting!")
+            return 0
```

Comment:
Should this be `1` for error? Or even better, raise an exception like `InvalidSpec`?

Reply:
Good idea!